libraries/Ion_auth.php

```diff
@@ -139,17 +139,21 @@ public function __get($var)
 	 */
 	public function forgotten_password($identity)
 	{
-		if ($this->ion_auth_model->forgotten_password($identity))
+		// Retrieve user information
+		$user = $this->where($this->ion_auth_model->identity_column, $identity)
```

Comment:
While I'm thinking about it, should probably add a default index on the default identify in the SQL files/migration.  Thoughts?

Reply:
Yes, as well as for both selector tokens (remember_me and password_reset).
I'll update the migration and user guide in another PR.

Reply:
However, in the SQL file, we don't know yet which column is the identity one, right?
Also, what happen if the user change the identity column option?
I think, for the identity column, it should remain as is with the info in the README (and maybe add a comment?) so users can implement optimization when needed.

Reply:
I think it'd be best to go ahead and optimize for the default identity and comment in the config that they should update their indexing if they change it.  In my experience the majority of users stick with the defaults.

Reply:
Alright, I'll do that.

Reply:
But in another PR 😅

Reply:
Yea sounds good, I'll go ahead and merge this one

Reply:
See PR #1210